pkg/cmd/server/api/types.go

```diff
@@ -144,6 +144,24 @@ type ProjectConfig struct {
 	// It is in the format namespace/template and it is optional.
 	// If it is not specified, a default template is used.
 	ProjectRequestTemplate string
+
+	// SecurityAllocator controls the automatic allocation of UIDs and MCS labels to a project. If nil, allocation is disabled.
```

Comment:
Just means the allocator is disabled.  We still need to define the security mode config, but it would be manual allocation vs automatic (perhaps by making the template have a common range and then moving projects out to their own blocks as needed)

> On Jun 8, 2015, at 8:03 AM, David Eads <notifications@github.com> wrote:
> 
> In pkg/cmd/server/api/types.go:
> 
> > @@ -144,6 +144,24 @@ type ProjectConfig struct {
> >  	// It is in the format namespace/template and it is optional.
> >  	// If it is not specified, a default template is used.
> >  	ProjectRequestTemplate string
> > +
> > +	// SecurityAllocator controls the automatic allocation of UIDs and MCS labels to a project. If nil, allocation is disabled.
> Disabled meaning nothing runs or disabled meaning everything runs as some user?
> 
> —
> Reply to this email directly or view it on GitHub.
>